dawn/main/ansible/Ansible.js

```diff
@@ -57,7 +61,7 @@ function buildProto(data) {
   return new DawnData({
     student_code_status: status,
     gamepads,
-    team_color: 0,
+    team_color: (stationNumber < 2) ? DawnData.TeamColor.BLUE : DawnData.TeamColor.GOLD,
```

Comment:
This `stationNumber` (as well as the others) should be given constants if possible.